apps/worker/src/app/workflow/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -196,6 +196,7 @@ export class SendMessageInApp extends SendMessageBase {
         transactionId: command.transactionId,
         content: this.storeContent() ? content : null,
         payload: messagePayload,
+        providerId: integration.providerId,
```

Comment:
funny thing, we did not save the provider on the message, which also means that the previous query of `oldMessage` always failed.